app/scripts/views/mixins/password-mixin.js

```diff
@@ -23,12 +23,15 @@ define(function (require, exports, module) {
       this.focus(controlsSelector);
     },
 
+  // Not needed anymore
```

Comment:
It's considered bad practice to leave code that is no longer needed in the tree. Can you remove this function from the source code? We can always go back through git history if we need the source again in the future.